conda/cli/common.py

```diff
@@ -281,3 +285,14 @@ def validate_prefix_is_writable(prefix: str) -> str:
     if isdir(dirname(test_path)) and file_path_is_writable(test_path):
         return prefix
     raise EnvironmentNotWritableError(prefix)
+
+
+def validate_env_file_exists(filename: str):
+    url_scheme = filename.split("://", 1)[0]
+    if url_scheme in CONDA_SESSION_SCHEMES:
```

Comment:
Should conda try to see if the remote file exists? It wasn't doing that previously. Maybe out of scope for this PR.